samples/subsys/ipc/ipc_service/remote/prj.conf

```diff
@@ -0,0 +1,34 @@
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_PRINTK=n
+CONFIG_IPM=y
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
+
+CONFIG_IPC_SERVICE=y
+CONFIG_IPC_SERVICE_BACKEND_RPMSG_MULTI_INSTANCE=y
+
+# Configuration backend for IPC Service
+CONFIG_IPM=y
+CONFIG_IPM_NRFX=y
```

Comment:
Like for host project, this file includes multiple entries specific for Nordic SoCs.

Reply:
Modified: Like for host sample.